Week5/app/src/main/java/com/em4n0101/mymovies/login/LoginActivity.kt

```diff
@@ -0,0 +1,51 @@
+package com.em4n0101.mymovies.login
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import com.em4n0101.mymovies.MainActivity
+import com.em4n0101.mymovies.R
+import com.em4n0101.mymovies.repositories.SharedPrefsRepository
+import kotlinx.android.synthetic.main.activity_login.*
+
+class LoginActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+
+        buttonSignIn.setOnClickListener {
+            if (validateFields())
+                navigateToMain()
+        }
+    }
+
+    private fun validateFields(): Boolean {
```

Comment:
If your methods return Boolean, try to name them so that the name indicates that. For example `isValid()` or `credentialsAreValid()`

Reply:
Also, it would be nice if you take validation logic outside of the activity, to a utility class or method.